### PR TITLE
Adds Assorted and SCP Themed flavors to gum and lollipops

### PR DESCRIPTION
**Pull request by chesse20**
more flavors

## About the Pull Request

image
dreamseeker_PhHAxm9MZg
Adds four colors and flavors of gum and lollipops, blue razberry, blackberry, and Euclid and Keter berry
## Why It's Good For The Game
Adds more SCP theming for immersion and makes rainbo-gum have more variation and more interesting for players to buy and use in-game.

## Changelog

:cl:
add: New flavors of gum and lolipops are now available! Can you collect and taste them all?
/:cl:

### CODE REVIEW

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Drinks.dm [hunk 1]
```diff
@@ -70,6 +88,14 @@
 	..()
 	M.reagents.add_reagent(/datum/reagent/medicine/imidazoline, removed * 0.2)
 
+/datum/reagent/drink/juice/euclid
+	name = "Eculid berry juice"
```
Comment (Vivalas) [inline code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Drinks.dm]: capitalization

Comment (Vivalas) [inline code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Drinks.dm]:
```suggestion
	name = "Euclid Berry Juice"
```

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Drinks.dm [hunk 2]
```diff
@@ -57,6 +57,24 @@
 	glass_name = "berry juice"
 	glass_desc = "Berry juice. Or maybe it's jam. Who cares?"
 
+/datum/reagent/drink/juice/blackberry
+	name = "Blackberry"
+	description = "Juice made from several thousand blackberries, probably."
+	taste_description = "blackberry"
+	color = "#8e0baf"
+
+	glass_name = "Blackberry juice"
```
Comment (Vivalas) [inline code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Drinks.dm]: On the otherhand, these should be lowercase to conform to the rest of the file
```suggestion
	glass_name = "blackberry juice"
```

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Drinks.dm [hunk 3]
```diff
@@ -57,6 +57,24 @@
...
+	glass_desc = "Its blackberry juice."
+
+/datum/reagent/drink/juice/blueraz
+	name = "Blue Razberry"
+	description = "An artifical fruit flavor created by man himself. "
+	taste_description = "blue razberry"
+	color = "#1408b8"
+
+	glass_name = "Blue Razberry Juice"
```
Comment (Vivalas) [inline code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Drinks.dm]: same here
```suggestion
	glass_name = "blue razberry juice"
```

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Drinks.dm [hunk 4]
```diff
@@ -57,6 +57,24 @@
 	glass_name = "berry juice"
 	glass_desc = "Berry juice. Or maybe it's jam. Who cares?"
 
+/datum/reagent/drink/juice/blackberry
+	name = "Blackberry"
```
Comment (Vivalas) [inline code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Drinks.dm]: add juice here to standardize with the rest of the file
```suggestion
	name = "Blackberry Juice"
```

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Drinks.dm [hunk 5]
```diff
@@ -57,6 +57,24 @@
...
+	description = "Juice made from several thousand blackberries, probably."
+	taste_description = "blackberry"
+	color = "#8e0baf"
+
+	glass_name = "Blackberry juice"
+	glass_desc = "Its blackberry juice."
+
+/datum/reagent/drink/juice/blueraz
+	name = "Blue Razberry"
```
Comment (Vivalas) [inline code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Drinks.dm:70]: juice
```suggestion
	name = "Blue Razberry Juice"
```

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Drinks.dm [hunk 6]
```diff
@@ -70,6 +88,14 @@
 	..()
 	M.reagents.add_reagent(/datum/reagent/medicine/imidazoline, removed * 0.2)
 
+/datum/reagent/drink/juice/euclid
+	name = "Eculid berry juice"
+	description = "Not safe enough for the normal population"
+	taste_description = "redacted"
+	color = "#FFFCFF" // rgb: 255, 140, 0
+
+	glass_name = "Euclid berry juice"
```
Comment (Vivalas) [inline code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Drinks.dm]: lowercase
```suggestion
	glass_name = "euclid berry juice"
```

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Drinks.dm [hunk 7]
```diff
@@ -70,6 +88,14 @@
...
+	glass_desc = "Safely contained in glass."
```
Comment (Vivalas) [inline code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Drinks.dm]:
```suggestion
	glass_desc = "Safely contained in a glass."
```

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Drinks.dm [hunk 8]
```diff
@@ -78,7 +104,14 @@
 
 	glass_name = "grape juice"
 	glass_desc = "It's grrrrrape!"
-
+/datum/reagent/drink/juice/keter
+	name = "Keter berry juice"
```
Comment (Vivalas) [inline code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Drinks.dm]:
```suggestion
	name = "Keter Berry Juice"
```

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Drinks.dm [hunk 9]
```diff
@@ -78,7 +104,14 @@
...
+	description = "A dangerous drink for dangerous people"
+	taste_description = "expunged"
+	color = "#EF1212" // rgb: 255, 140, 0
+
+	glass_name = "Keter berry juice"
```
Comment (Vivalas) [inline code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Drinks.dm]:
```suggestion
	glass_name = "keter berry juice"
```

**Review by Vivalas — CHANGES_REQUESTED** [review]

**Comment by chesse20** [comment]
i merged the right suggested fixes i think

**Comment by Vivalas** [comment]
Euclid berry juice and Keter berry juice still need to be fully capitalized, i.e. "Keter Berry Juice" "Euclid Berry Juice" in the reagent name. The glass_names are correct.

**Comment by chesse20** [comment]
> Euclid berry juice and Keter berry juice still need to be fully capitalized, i.e. "Keter Berry Juice" "Euclid Berry Juice" in the reagent name. The glass_names are correct.

got it 😄

**Review by Vivalas — APPROVED** [review]

**Pull request merged**